content/terraform/terraform.tf

```diff
@@ -93,20 +93,5 @@ module "content" {
   secondary_container_port           = "3000"
   host_name                          = "content.wellcomecollection.org"
   healthcheck_path                   = "/management/healthcheck"
-  alb_priority                       = "002"
-}
-
-resource "aws_alb_listener_rule" "articles_path_rule" {
-  listener_arn = "${local.alb_listener_https_arn}"
-  priority     = "004"
-
-  action {
-    type             = "forward"
-    target_group_arn = "${module.content.target_group_arn}"
-  }
-
-  condition {
-    field  = "path-pattern"
-    values = ["/exhibitions*"]
-  }
+  alb_priority                       = "700"
```

Comment:
This isn't really right.
@wellcometrust/experience-dev let's figure some logic out around this.